modules/custom/az_person/config/install/field.field.node.az_person.field_az_email.yml

```diff
@@ -9,7 +9,7 @@ field_name: field_az_email
 entity_type: node
 bundle: az_person
 label: Email
-description: ''
+description: 'If possible, please use addresses ending with <i>@arizona.edu</i> rather than <i>@email.arizona.edu</i>.'
```

Comment:
```suggestion
description: 'If possible, <a href="https://uaatwork.arizona.edu/uannounce/send-arizonaedu-email-effective-march-10">please use addresses ending with <i>@arizona.edu</i> rather than <i>@email.arizona.edu</i></a>.'
```